rosdoc2/verbs/build/builders/doxygen_builder.py

```diff
@@ -152,6 +163,16 @@ def __init__(self, builder_name, builder_entry_dictionary, build_context):
 
     def build(self, *, doc_build_folder, output_staging_directory):
         """Actually do the build."""
+        # If the build type is not 'ament_cmake/cmake', there is no reason to run doxygen.
+        if (
+            self.build_context.build_type not in
+            ('ament_cmake', 'cmake') and not self.build_context.always_run_doxygen
+        ):
+            logger.info(
+                f"The package build type is not 'ament_cmake' or 'cmake', hence the"
+                f"'{self.builder_type}' builder was not invoked")
```

Comment:
```suggestion
            logger.debug(
                f"The package build type is not 'ament_cmake' or 'cmake', hence the "
                f"'{self.builder_type}' builder was not invoked")
```

Reply:
Addressed in https://github.com/ros-infrastructure/rosdoc2/pull/28/commits/7d0b4fd4dff3ca0c92f8364ab39b47625f944dbf